apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java

```diff
@@ -15,10 +16,11 @@
 import retrofit2.converter.moshi.MoshiConverterFactory;
 
 public class SampleApplication extends Application {
-  private static final String BASE_URL = "https://githunt-api.herokuapp.com";
+  // Change localhost to your machine's local IP address when running from a device
+  private static final String BASE_URL = "http://10.6.1.232:8080";
```

Comment:
Ah, this wasn't an explicit decision so much as I copied the behavior from the `frontpage-ios-app`. the `10.x.x.x` IP is my mistake in checking in my local IP (the ios app has checked in `127.0.0.1`)